app/code/core/Mage/Contacts/etc/system.xml

```diff
@@ -89,6 +89,23 @@
                         </email_template>
                     </fields>
                 </email>
+                <security translate="label">
+                    <label>Security</label>
+                    <sort_order>60</sort_order>
+                    <show_in_default>1</show_in_default>
+                    <show_in_website>1</show_in_website>
+                    <show_in_store>1</show_in_store>
+                    <fields>
+                        <enable_form_key translate="label comment">
```

Comment:
Since in "core" we have use_form_key:
<img width="242" alt="Schermata 2022-07-30 alle 18 40 52" src="https://user-images.githubusercontent.com/909743/181935251-cfb58159-60ac-46b3-800e-376608de5533.png">
wouldn't it be better to use the same?

I was also thinking that we had a single flag to enable csfr for all openmage:
<img width="426" alt="Schermata 2022-07-30 alle 18 41 59" src="https://user-images.githubusercontent.com/909743/181935290-fd996043-fd9a-4c32-9a67-f27ed340c611.png">
shouldn't we use that single one instead of creating a flag for each module?

Reply:
Yes it would be better to have one global flag, but a separate one is added for backwards-compatibility because if users have `system/csrf/use_form_key` enabled, their contacts form won't work if they don't apply the change to their custom themes.

Reply:
I thought so but at the same time now they update and they don't have the new security feature enabled so I don't know. Also the idea of having new configuration options for the same thing I don't know :-\

Reply:
I'm not a big fan of it either but I can't think of another way to fix this issue. We could remove these separate flags in `20.0`(or superior versions) and keep only the global one. There's also a separate flag for the checkout I think.

Reply:
I think a major reason why many features, especially security related, go unnoticed is because users don't have an easy way of knowing about them. It is easy for such changes to be lost in the changelog next to tens of other changes. Using the notifications for this purpose would be good, but I remember seeing opposing opinions about that in some thread.